### ISSUE TITLE
Unable to tap delete confirmation button is table view.

### ISSUE THREAD

**Issue by scraton** [kif-framework/KIF#98]
I have the following steps defined in a scenario testing a table view:

``` objc
    [scenario addStep:[KIFTestStep stepToTapViewWithAccessibilityLabel:@"Edit" traits:UIAccessibilityTraitButton]];
    [scenario addStep:[KIFTestStep stepToTapViewWithAccessibilityLabel:@"Delete McDonalds" traits:UIAccessibilityTraitButton]];
    [scenario addStep:[KIFTestStep stepToTapViewWithAccessibilityLabel:@"Confirm Deletion for McDonalds" traits:UIAccessibilityTraitButton]];
```

The first two steps will pass without issue (assuming the table view is properly configured). The problem is the third step. It cannot find the confirmation deletion control (these are all standard iOS controls, no customization). Accessibility is enabled for the item, but after digging through the code, the problem seems to be that the `hitTest:withEvent:` method for the coordinates will always return a `UIViewControllerWrapperView` object rather than the button control itself.

It's important to note that it _can_ find the button by accessibility label. The proper `UIAccessibilityElement` is found. But when it tries to determine if it `isTappable`, it will perform a `hitTest:` on the window and always return that wrapper view class instead of the button control.

I've chocked it up to probably being some security protection on iOS's part so that you can't automate the deletion of items, but perhaps y'all can dig deeper and figure out what is really going on. Or perhaps think of a workaround by overriding some private API's.
